android/src/oldarch/CleverTapModule.kt

```diff
@@ -524,6 +523,88 @@ class CleverTapModule(reactContext: ReactApplicationContext?) :
         cleverTapModuleImpl.clearInAppResources(expiredOnly)
     }
 
+    @ReactMethod
+    fun customTemplateSetDismissed(templateName: String?, promise: Promise?) {
+        cleverTapModuleImpl.customTemplateSetDismissed(templateName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateSetPresented(templateName: String?, promise: Promise?) {
+        cleverTapModuleImpl.customTemplateSetPresented(templateName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateRunAction(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateRunAction(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateGetStringArg(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateGetStringArg(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateGetNumberArg(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateGetNumberArg(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateGetBooleanArg(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateGetBooleanArg(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateGetFileArg(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateGetFileArg(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateGetObjectArg(
+        templateName: String?,
+        argName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateGetObjectArg(templateName, argName, promise)
+    }
+
+    @ReactMethod
+    fun customTemplateContextToString(
+        templateName: String?,
+        promise: Promise?
+    ) {
+        cleverTapModuleImpl.customTemplateContextToString(templateName, promise)
+    }
+
+    @ReactMethod
+    fun syncCustomTemplates() {
+        cleverTapModuleImpl.syncCustomTemplates()
+    }
+
+    @ReactMethod
+    fun syncCustomTemplatesInProd(isProduction: Boolean) {
+        cleverTapModuleImpl.syncCustomTemplates()
```

Comment:
isProd flag is not used here
Is that intentional?

Reply:
Yes, this methods is only used in iOS for Test Flight builds, but it has to be implemented in the Android module as well. On Android it would do the same as syncCustomTemplates.